src/components/SafeDepositRental/RenderRadioButtons.tsx

```diff
@@ -0,0 +1,28 @@
+import { JSX } from '@redneckz/uni-jsx';
+import { useState } from '@redneckz/uni-jsx/lib/hooks/core';
+import { Radio } from '../../ui-kit/Radio/Radio';
+
+export const RenderRadioButtons = JSX(() => {
+  const [selected, setSelected] = useState('option1');
```

Comment:
```suggestion
  const [selectedOption, setSelectedOption] = useState('option1');
```

Иначе воспринимается как булевое. Не уверен, что это лучшее значение по умолчанию